src/qililab/platform/platform.py

```diff
@@ -576,11 +576,34 @@ def execute_qprogram(  # pylint: disable=too-many-locals
     ) -> QProgramResults:
         """Execute a QProgram using the platform instruments.
 
+        |
+
+        **The execution is done in the following steps:**
+
+        1. Compile the QProgram.
+        2. Upload the sequences to the instruments.
+        3. Execute the sequences.
+        4. Acquire the results.
+        5. Reset the instrument settings.
+
+        |
+
+        **The execution is done in the following way:**
+
+        - If all the buses in the QProgram are associated with Qblox modules, the execution is done using the QbloxCompiler.
+        - If all the buses in the QProgram are associated with Quantum Machines Clusters, the execution is done using the QuantumMachinesCompiler.
+        - If the buses in the QProgram are associated with a mixture of Qblox modules and Quantum Machines Clusters, the execution is not supported.
+
         Args:
             qprogram (QProgram): The QProgram to execute.
+            bus_mapping (dict[str, str], optional): A dictionary mapping the buses in the QProgram to the buses in the platform.
```

Comment:
The statement is correct but the concepts are 2 sentences that repeat each other.
bus_mapping (dict[str, str], optional): A dictionary mapping the buses in the QProgram (keys) to the buses in the platform (values). It is useful for mapping specific experiments to generic Qprograms. Defaults to None.
something like this?

Reply:
I like it, changed it!
Thanks 🔥